src/node_file.cc

```diff
@@ -3130,22 +3130,34 @@ static void CpSyncCheckPaths(const FunctionCallbackInfo<Value>& args) {
   ToNamespacedPath(env, &src);
   THROW_IF_INSUFFICIENT_PERMISSIONS(
       env, permission::PermissionScope::kFileSystemRead, src.ToStringView());
+
+#ifdef _WIN32
+  auto src_path = std::filesystem::path(src.ToU8StringView());
+#else
   auto src_path = std::filesystem::path(src.ToStringView());
+#endif
+
+  std::cout << src.ToStringView() << std::endl;
```

Comment:
Are these debug logs? They should be left out of the PR.

Reply:
There are segfault issues with the fix happening on CI but not my local Windows machine. I am having these debug lines in case to navigate the issue on CI. Please see [this thread](https://github.com/nodejs/build/issues/3901) for more context.